app.js

```diff
@@ -1,44 +1,12 @@
 require('dotenv').config();
+const personaRouter = require('./routes/persona');
 const express = require('express');
-const connectDB = require('./db');
-connectDB();
 const app = express();
-const port = 3000
-const Persona = require('./models/persona');
+const port = 3000;
```

Comment:
esto para ya tenerlo listo para cualquier ambiente y no `3000` por defecto

```suggestion
const port = process.env.PORT || 3000;
```